prometheus_client/metrics.py

```diff
@@ -310,6 +316,11 @@ def inc(self, amount: float = 1, exemplar: Optional[Dict[str, str]] = None) -> N
             _validate_exemplar(exemplar)
             self._value.set_exemplar(Exemplar(exemplar, amount, time.time()))
 
+    def reset(self) -> None:
+        """Reset the metric to zero."""
```

Comment:
It would be nice to have some of the context around logical process restarts here as well. Perhaps
```suggestion
        """Reset the counter to zero. Use this when a logical process restarts without restarting the actual python process."""
```
?